docs/schema/constants/line-join.json

```diff
@@ -3,18 +3,23 @@
     "type": "integer",
     "title": "LineJoin",
     "description": "",
-    "oneOf": [
-        {
-            "title": "Miter",
-            "const": 1
-        },
-        {
-            "title": "Round",
-            "const": 2
-        },
-        {
-            "title": "Bevel",
-            "const": 3
+    "default": 2,
+    "properties": {
+        "lj": {
```

Comment:
Why switching this from an enum to a property?

Reply:
Because then we can maintain the same values at one place and reuse them as a `$ref`. For example, I've included line-join into these 2 files:
1.  https://github.com/LottieFiles/lottie-docs/pull/10/files#diff-6842843da38497db2665d2144dd1dfb6a991303234eabfc1070497aa1872401dR8
2. https://github.com/LottieFiles/lottie-docs/pull/10/files#diff-3cf937f1c26d726406b5d69a6f41cd0bbabe5905669c79bd070819a8d7f84291R11

but ofc I can keep only the const value in the file instead of moving the entire property.

Reply:
I think it's better to have `lj` in the objects and `$ref` pointing to only the values